media/js/mozorg/about-transparency.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, Mozilla) {
+    'use strict';
+
+    var reportSections = $('#report-sections');
+
+    if (reportSections) {
+        if (typeof matchMedia !== 'undefined') {
+
+            var queryWide = matchMedia('(min-width: 760px)');
+
+            if (queryWide.matches) {
+                Mozilla.Accordion.destroyAccordionById('report-sections');
+            } else {
+                reportSections = new Mozilla.Accordion(reportSections);
```

Comment:
Now see, jshint complained about that ("Do not use 'new' for side effects.") but eslint is fine with it. I never know who to trust...